README.md

```diff
@@ -58,28 +58,34 @@ If you would like to experiment with running this code against LabKey Server it
 
 #### Deploy Using Experimental Build
 
+:warning: This will modify files in your git enlistment. Double check your commits and be sure to revert and changes when you are done experimenting!
```

Comment:
```suggestion
:warning: This will modify files in your git enlistment. Double check your commits and be sure to revert any changes when you are done experimenting!
```

Reply:
Done.